### DISCUSSION TITLE
Why it always show AUTH_KEY_UNREGISTERED

### DISCUSSION THREAD

**Discussion by cmingou** [xelaj/mtproto#74] [Q&A] [answered]
After I execute auth example and input the code in SMS. I believe I got the right session.json.
When I execute other example like get chat or else, it always got error

> error: &errors.withMessage{
    cause: &mtproto.ErrResponseCode{
      Code:           401,
      Message:        "AUTH_KEY_UNREGISTERED",
      Description:    "The key is not registered in the system",
      AdditionalInfo: nil,
    },
    msg: "sending AccountGetContentSettings",
  }

Is there any wrong step I did?

**Comment by quenbyako** [comment, marked as answer]
Het @cmingou! this issue (#65) fixed already, fetch latest commit and try it :smiley:

**Reply by cmingou** [reply]
I tried commit 5ba8d1f663ee.
After I delete session.json and execute auth again it work like normal.
When I execute another example or execute auth again, it's seems the session is not registred and still show 'AUTH_KEY_UNREGISTERED'

**Reply by quenbyako** [reply]
@cmingou 
1) delete example session. usually (if you didn't something customized and just tried to run examples) it located at `~/.local/var/lib/mtproto-example/session.json`
2) be sure that you has non-changed latest version: delete all package from `$GOPATH/src/github.com/xelaj/mtproto` fetch again with `go get -v -u github.com/xelaj/mtproto` (or go into package and tap `git status`)
3) run again `go run ./examples/auth` and authorize your fresh new session
4) then try to run another examples like `go run ./examples/get_self`

**Reply by cmingou** [reply]
I found where is the problem.
I forgot to change the apiID and apiHash in AuthSendCode...
All things work. Thanks.

B.T.W. It seems some function in API document cannot be found implementation.
Which version of API already mapping?
